api/nakadi-event-bus-api.yaml

```diff
@@ -1250,6 +1250,8 @@ definitions:
 
           To have a generic, undefined schema it is possible to define the schema as `"schema":
           "{\"additionalProperties\": true}"`.
+
+          If type of schema is JSON_SCHEMA then schema can be passed as json object for convenience.
```

Comment:
It seems a better option for EventTypeSchema to have mandatory field type and then use it as a `discriminator`, so other fields are defined on their own definitions `json_schema`  and `json_schema_object`.
The conversion between schema types in general not always possible, so if it was defined like a json_schema then it should stay a string, if it was json_schema_object then it should stay object to avoid confusion.